examples/billboard/README.md

```diff
@@ -7,4 +7,4 @@ If you are cost conscious and need to control your environment, then you can use
 
 ## Run the walkthrough tutorial
 
-[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2Fayapsone%2Fbillboard&cloudshell_tutorial=billboard-walkthrough.md)
+[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/ayapsone/professional-services&cloudshell_tutorial=examples/billboard/billboard-walkthrough.md)
```

Comment:
Why is the button cloning the professional-services repo in your github ? Should this point to this repo ?